_use_cases/encryption-at-rest.markdown

```diff
@@ -0,0 +1,105 @@
+---
+name: Encryption At Rest
+---
+
+## Why
+
+Apache Kafka&#174; does not directly support any form of encryption for data stored within a broker. This means that the contents
+of records sent to Apache Kafka are stored in the clear. Anyone with sufficient privileges, such as a Kafka Administrator
+with file system access, is able to read the contents of the records.
+
+This present a challenge for an enterprise using Kafka to distribute confidential data or PII (Personally Identifiable
```

Comment:
```suggestion
This presents a challenge for an enterprise using Kafka to distribute confidential data or PII (Personally Identifiable
```

Also it's not "confidential data or PII": PII is a subset of confidential data, so "confidential data, such as PII"... would be better imho.